js/utils/urlUtils.ts

```diff
@@ -14,5 +14,5 @@ export const isLocal = () => {
 };
 
 export const getSykefravaerarbeidsgiverUrl = (sti = '') => {
-  return isLabs() ? process.env.SYKEFRAVAERARBEIDSGIVER_LABS_URL : sti;
+  return isLabs() ? process.env.SYKEFRAVAERARBEIDSGIVER_LABS_URL || '' : sti;
```

Comment:
Hm, kan dette skjule feil dersom process.env ikke funker / path er feil?

Reply:
Vi bør sikkert ta en process.env oppvasksoppgave :p

Reply:
Jepp, lagde en task på det: https://trello.com/c/WP1LXiLg/651-refaktorere-bruk-av-env-fil-environmnet-variables-etc

Jeg lurer på om appen potensielt kræsjer om den ikke finner env-var'en, var det ikke det som skjedde sist vi fjernet en env-var som ble brukt? `|| ''` er for at ts ikke skal interprete at funksjonen kan returnere `undefined`.